CollectAudioLogs/audio-info.wprp

```diff
@@ -0,0 +1,861 @@
+<?xml version="1.0" standalone="yes"?>
+<WindowsPerformanceRecorder Version="1.0" Author="" Team="" Comments="" Company="Microsoft Corporation" Copyright="Microsoft Corporation" Tag="">
+  <Profiles>
+    <!-- Collector for system tracing -->
+    <SystemCollector Id="SystemCollector" Name="System Collector">
+      <BufferSize Value="256" />
+      <Buffers Value="3" PercentageOfTotalMemory="true" MaximumBufferSpace="256" />

```

Comment:
Maybe have the MaximumBufferSpace set to 128 for all 3? That way we get more audio info, and this trace has significantly less system provider collection, so having all 3 at ~128 will probably give a more balanced log.

As it is, with the 256/64/64 split in audio-glitches-manual, we often lose audio logging to the circular buffer, but retain the system information logging for the entire duration of collection by the user.

Reply:
I'm pulling this .wprp from windows.media.common, so we should discuss any changes to the .wprp that may be useful in a separate change